### PR TITLE
New tip : converting a node list to an array

### PR DESCRIPTION
**Pull request by tevko**
A safe, easy, and functional way to convert a node list into an array

### CODE REVIEW

**Comment by johnstew** [comment]
+1

**Comment by manmadareddy** [comment]
:+1:

**Comment by loverajoel** [comment]
@tevko nice tip!
This will be the tip for tomorrow, please make a rebase and update the number post and the date!
Thanks for your tip!

**Comment by tevko** [comment]
@loverajoel ok I think I did this correctly

**Comment by hingsir** [comment]
@tevko `Array.apply` has higher performance than `[].slice.apply` or `Array.prototype.slice.apply`.  Is that right?

File: README.md [hunk 1]
```diff
@@ -53,22 +74,23 @@ By including this directive in a JavaScript file or function, we will direct the
...
 * Firefox from version 4.
-* Chrome from version 13. 
-* Safari from version 5.1. 
+* Chrome from version 13.
+* Safari from version 5.1.
 * Opera from version 12.
 
 [See MDN for a fuller description of strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode).
-
+=======
```
Comment (loverajoel) [inline README.md]: remove this

**Comment by tevko** [comment]
@loverajoel fixed

**Comment by loverajoel** [comment]
@hingsir I'm not sure, How we can confirm or test that?

**Comment by loverajoel** [comment]
@tevko thanks for your tip!

**Comment by sergiodxa** [comment]
Why not just `Array.from(NodeList)`?

**Comment by tevko** [comment]
@sergiodxa browser support

**Comment by loverajoel** [comment]
@tevko https://github.com/loverajoel/jstips/issues/53

**Pull request merged**